docs/api/reports.md

```
@@ -0,0 +1,400 @@
+      show_root_heading: true
+      show_source: false
+      members:
+        - __init__
+        - generate
+        - save
+
+### Usage
+
+=== "Generate and save"
+
+    ```python
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)